src/init.py

```diff
@@ -36,8 +36,9 @@
     # If a mixture is used, the number of each compound in the mixture
     # needs to be specified:
     # "n_compounds" = [(100,100), (1000,500)]
```

Comment:
```suggestion
    # "n_compounds" = [[100,100], [1000,500]]
```
does this need to be list instead of tuple?

Reply:
Yeah, I fixed the doc strings